universum/analyzers/utils.py

```diff
@@ -2,13 +2,28 @@
 import sys
 
 
-def add_common_arguments(parser):
+def add_files_argument(parser):
```

Comment:
If it is only for python files, the function name must reflect it.

Reply:
No, the descripion is wrong (copied from pylint). Files argument is common for analyzers (e.g. uncrustify has it too)
However, I just noticed that destination for uncrustify's --files argument is file_names instead of file_list. I'll change the destination variable name

Reply:
should be fixed in new revision